tests/conftest.py

```diff
@@ -11,10 +12,19 @@
 from stripe.six.moves.urllib.error import HTTPError
 
 from tests.request_mock import RequestMock
+from tests.stripe_mock import StripeMock
 
 
 MOCK_MINIMUM_VERSION = '0.40.0'
-MOCK_PORT = os.environ.get('STRIPE_MOCK_PORT', 12111)
+if StripeMock.start():
```

Comment:
Should've noticed this in stripe-ruby too probably, but it might be worthwhile to have a succinct informational comment here to the tune of:

> Starts stripe-mock if an OpenAPI spec override is found in `openapi/`, and otherwise falls back to `STRIPE_MOCK_PORT` or 12111.